CONTRIBUTING.md

```diff
@@ -0,0 +1,11 @@
+# CONTRIBUTING
+
+1. Write a comment on the issue you want to work on.
```

Comment:
We could add that we have a "good first issue" label to get started contributing on this repo, but since it's a common practice, it not mandatory